### PR TITLE
Scene Audio Settings

### PR DESCRIPTION
**Pull request by robertlong**
You can now adjust the audio settings for avatars and media in your scene. Select the scene node in the hierarchy panel and check "Override Audio Settings" to adjust these properties.

Screen Shot 2020-02-12 at 11 16 11 AM

### CODE REVIEW

File: .env.defaults [hunk 1]
```diff
@@ -7,10 +7,10 @@
 # The Hubs backend to use for previewing published scenes.
 # The RETICULUM_SERVER must also be set to dev to avoid the CSP rules.
 # Uncomment both lines when testing with a local hubs instance.
-# HUBS_SERVER="localhost:8080"
+HUBS_SERVER="localhost:8080"
```
Comment (gfodor) [inline .env.defaults]: intentional?

File: src/ui/properties/SceneNodeEditor.js [hunk 2]
```diff
@@ -23,90 +27,245 @@ const FogTypeOptions = [
...
+          name="Fog Density"
+          smallStep={0.01}
+          mediumStep={0.1}
+          largeStep={0.25}
+          min={0}
+          value={node.fogDensity}
+          onChange={onChangeFogDensity}
+        />
+      )}
+      <InputGroup name="Override Audio Settings">
```
Comment (gfodor) [inline src/ui/properties/SceneNodeEditor.js]: Perhaps add `info` with a link to the pannernode settings page on MDN?

Comment (robertlong) [reply src/ui/properties/SceneNodeEditor.js]: Info is a tooltip, so you can't actually click on links in it. Changing the UX of prop info seems out of scope for this PR. However, I agree that would be a nice thing to add.

File: .env.defaults (hunk 1 above)
Comment (robertlong) [reply .env.defaults]: Nope, thanks!

**Pull request merged**